lib/libuutil/uu_dprintf.c

```diff
@@ -76,14 +76,10 @@ uu_dprintf_create(const char *name, uu_dprintf_severity_t severity,
 	if ((D = uu_zalloc(sizeof (uu_dprintf_t))) == NULL)
 		return (NULL);
 
-	if (name != NULL) {
-		D->uud_name = strdup(name);
-		if (D->uud_name == NULL) {
-			uu_free(D);
-			return (NULL);
-		}
-	} else {
-		D->uud_name = NULL;
+	D->uud_name = strdup(name);
+	if (D->uud_name == NULL) {
+		uu_free(D);
+		return (NULL);
```

Comment:
@behlendorf
thanks for your suggest
commit other PR:https://github.com/zfsonlinux/zfs/pull/5376
please review it.
thanks.